subprojects/tooling-api/src/main/java/org/gradle/tooling/internal/adapter/ProtocolToModelAdapter.java

```diff
@@ -232,7 +233,7 @@ public static String toWords(CharSequence string, char separator) {
             if (builder.length() > 0) {
                 builder.append(separator);
             }
-            String group1 = matcher.group(1).toLowerCase();
+            String group1 = matcher.group(1).toLowerCase(Locale.ENGLISH);
```

Comment:
Using `TextUtil` here breaks backward compatibility.

Reply:
Should we be using the `US` locale instead? Gradle uses US English for its messages.